worldengine/cli/main.py

```diff
@@ -391,8 +391,8 @@ def main():
     if args.seed is not None:
         seed = int(args.seed)
     else:
-        seed = random.randint(0, 65535)#RNG initialization is done automatically
-    random.seed(seed)
+        seed = numpy.random.randint(0, 999999)  # RNG initialization is done automatically
```

Comment:
Ok, I added this in to be safe. It would be good, though, if we could add a very clear comment about this. (Or just remove the restriction if there is none.)